.eslintrc.json

```diff
@@ -0,0 +1,162 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2022,
+        "project": [
+            "./tsconfig.eslint.json"
+        ],
+        "sourceType": "module"
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
```

Comment:
Not long ago I updeted this to plugin:@typescript-eslint/strict-type-checked
It has some more useful rules. You can do it as a separate PR if you want

Reply:
Also with that some of the items in the "extends" list are not needed. You can check out latest cli repo to see